### DISCUSSION TITLE
Current description for an Aspect not visible when editing

### DISCUSSION THREAD

**Discussion by GerlofdeHaan** [alkem-io/alkemio#563] [General] [answered]
The current description (when created) is not visable when you start editing the Aspect via "Settings".

**Comment by Comoque1** [comment]
Hi Gerolf, thanks for the feedback! 
Bug is raised in our development backlog and will be prioritized accordingly.

**Comment by Comoque1** [comment, marked as answer]
Hi Gerolf, the fix is applied on production, with the deployment from today.

Regards!
